src/charm.py

```diff
@@ -36,7 +36,7 @@ class WazuhServerCharm(CharmBaseWithState):
     """Charm the service.
 
     Attributes:
-        fqdns: the unit FQDNs.
+        master_fqdn: the FQDN for unit 0.
```

Comment:
nit: might pose a problem with woke

Reply:
Already ignored. This is Wazuh's terminology, so it's fine